### ISSUE TITLE
Qiskit version section elements in Homepage are not sized appropriately

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/qiskit.org#3197]
### Feature description

A Lighthouse reported showed that the elements in the Qiskit version section in the homepage (in the hero) include interactive elements that don’t have enough space around them, so that they could be easy to tap without overlapping.

We need to improve the layout of that section of the page.

This issue affects the UX, accessibility and SEO.

### Additional context

This is the section:

![Screenshot 2023-05-09 at 17 30 50](https://github.com/Qiskit/qiskit.org/assets/22047320/a82fe06c-0223-452a-8e3a-7fcb53739cfe)

**Comment by techtolentino** [comment]
@y4izus I can't seem to find this violation when running a Lighthouse scan, are you able to see the warning when you do it? (Since Eddy mentioned the word "tap" above, I'm assuming that this is a mobile issue.)

Otherwise, I can simply add spacing between the links or modify the arrangement of the CTAs so that we know for sure it's accessible.

What are your thoughts?

Something like this could be an improvement to the spacing:

![Image](https://github.com/Qiskit/qiskit.org/assets/6276074/570559fa-6150-44da-8a28-1362576a5e06)

**Comment by techtolentino** [comment]
@eddybrando 

With a redesign coming - should we even work on this?

**Comment by eddybrando** [comment]
This is the issue from the report generated in https://pagespeed.web.dev/analysis/https-qiskit-org/s9aww60d03?form_factor=mobile:

![Screenshot 2023-07-31 at 15 45 20](https://github.com/Qiskit/qiskit.org/assets/22047320/ee07c265-cb03-42e5-99a7-f6b5ae02c219)

This is the linked page with more information about this kind of issue: https://developer.chrome.com/docs/lighthouse/seo/tap-targets/?utm_source=lighthouse&utm_medium=lr

**Comment by eddybrando** [comment]
> @eddybrando
> 
> With a redesign coming - should we even work on this?

This is a question for @y4izus

**Comment by y4izus** [comment]
This issue is not relevant anymore, so I close it
